tests/integrations/config/session.py

```diff
@@ -4,4 +4,13 @@
 DRIVERS = {
     "default": "cookie",
     "cookie": {},
+    "redis": {
+        "driver_config": {
```

Comment:
Why are we wrapping this inside a driver_config dictionary? Can' we just do `host`, `port`, `password` inside the `redis` connection? This looks ugly to me and not really similar to other features

Reply:
The reason for this is that if you do this in the same way as per the Redis Cache Driver there is no easy way to provide additional initial configuration to the underlying connection driver StrictRedis.
 ie how do I set the "db=3" on the connection or enable any of the ssl options?
There is no facility for this at all nor on any of the other drivers that I could tell 
In the case of the StrictRedis driver there are 35+ options available to configure the driver.

This  might cause the developer for example to write a subclass of the RedisDriver to override the get_connection() method to use out of the box  options that the driver module provides. 

Yes it is a bit ugly I admit but I could not see a better way to describe the options that would be applied to the module driver itself.
While still providing configuration to the wrapper (session in this case) for things like timeout and namespace.  
Putting all module config and wrapper config options at the same level means you then have to separate them at the wrapper level by some other means.   

Ideally the framework should provide the ability to use the underlying module the ServiceProvider wraps with any configuration that is defined by the underlying module/s it uses in as easy a manner as possible. 
Not force the developer  to write additional code to handle something the underlying module (driver) already provides.

I hope this clarifies the design choice.

Happy to discuss more about better ways to provide this functionality

Reply:
Maybe the solution is to do something like this:

```python
DRIVERS = {
    #...
    "redis": {
        "host": "127.0.0.1",
        "port": "6379",
        "password": "",
        "options": {},
        "namespace": "",
        "timeout": "",
    }
}
```
and then build the connection like this:

```python
self.connection = Redis(
    **self.options.get("options"),
    host=self.options.get("host"),
    port=self.options.get("port"),
    password=self.options.get("password"),
)
```

(It's what is used for Masonite ORM config)

Reply:
@girardinsamuel  thanks thats much cleaner and still provides flexibility.
Implemented it and its working ;)